tests/test_kaggle_cache_competition_download.py

```diff
@@ -0,0 +1,66 @@
+import os
+from unittest import mock
+
+import requests
+
+import kagglehub
+from kagglehub.config import DISABLE_KAGGLE_CACHE_ENV_VAR_NAME
+from kagglehub.env import KAGGLE_DATA_PROXY_URL_ENV_VAR_NAME
+from tests.fixtures import BaseTestCase
+
+from .server_stubs import jwt_stub as stub
+from .server_stubs import serv
+
+INVALID_ARCHIVE_COMPETITION_HANDLE = "invalid/invalid/invalid/invalid/invalid"
```

Comment:
Nit, since an input would only be invalid if there is a "/" maybe having multiple here is overkill? Up to you if it should 1 or more

Reply:
lol thx